Dockerfile

```diff
@@ -22,4 +22,3 @@ RUN addgroup -S errbot \
 
 USER errbot
 WORKDIR /opt/errbot
-CMD errbot
```

Comment:
Why is this removed? `CMD` doesn't force you to run the command unlike `ENTRYPOINT`